### PR TITLE
Switch to containers/common for seccomp

### PR DESCRIPTION
**Pull request by saschagrunert**
#### What type of PR is this?

/kind cleanup

#### What this PR does / why we need it:
seccomp/containers-golang is deprecated and we should switch to
containers/common/pkg/seccomp instead.
#### How to verify it
Check if the CI passes.
#### Which issue(s) this PR fixes:

None
#### Special notes for your reviewer:
None
#### Does this PR introduce a user-facing change?

```release-note
None
```

### CODE REVIEW

**Comment by rhatdan** [comment]
LGTM

File: go.mod [hunk 1]
```diff
@@ -4,10 +4,10 @@ go 1.12
 
 require (
 	github.com/containernetworking/cni v0.7.2-0.20190904153231-83439463f784
-	github.com/containers/common v0.20.3
+	github.com/containers/common v0.20.3-0.20200827091701-a550d6a98aa3
```
Comment (TomSweeneyRedHat) [inline go.mod:7]: @rhatdan just bumped to v0.21.0, would you mind changing this to that please @saschagrunert ?

Comment (saschagrunert) [reply go.mod]: Sure, done :+1:

**Comment by TomSweeneyRedHat** [comment]
Thanks for the bump @saschagrunert 
bors r+

**Comment by TomSweeneyRedHat** [comment]
LGTM

**Comment by nalind** [comment]
bors r=TomSweeneyRedHat

**Pull request merged**